soh/soh/Enhancements/cosmetics/CosmeticsEditor.cpp

```diff
@@ -125,6 +126,8 @@ static std::map<std::string, CosmeticOption> cosmeticOptions = {
     
     COSMETIC_OPTION("Equipment_BoomerangBody",       "Boomerang Body",       BOX_EQUIPMENT,    ImVec4(160, 100,   0, 255), false, true, false),
     COSMETIC_OPTION("Equipment_BoomerangGem",        "Boomerang Gem",        BOX_EQUIPMENT,    ImVec4(255,  50, 150, 255), false, true, true),
+    COSMETIC_OPTION("Equipment_SlingshotBody",       "Slingshot Body",       BOX_EQUIPMENT,    ImVec4(160, 100,   0, 255), false, true, true),
+    COSMETIC_OPTION("Equipment_SlingshotString",     "Slingshot String",     BOX_EQUIPMENT,    ImVec4(255, 255, 255, 255), false, true, true),
```

Comment:
Tiny nitpick on the name, wouldn't it be better to be SlingshotChain ?

Reply:
Are you thinking of hookshot chain? It would be a string for a slingshot right?

Reply:
> Are you thinking of hookshot chain? It would be a string for a slingshot right?

omygosh... I brainfarted hard on that one, don't mind me, i'm an idiot 
*hide in the corner of shame*

Reply:
Happens to the best of us 😆